### DISCUSSION TITLE
Add Metal Barrels

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#205] [Ideas]
Because if you have too many chests around, the fps of the game will decrease, but Barrels don't cause this problem (add a Move Barrel tool as well).

**Comment by Swifty07** [comment]
this isnt vannilla

**Comment by GregoriusT** [comment]
It would be IronChests Compat in that case though ;)

**Comment by Roadhog360** [comment]
Iron shulker boxes are a backport, this would not be.

**Comment by DarianLStephens** [comment]
What are the iron shulkers, in that case?
A backport of Iron Chests stuff? Because there are mods for Iron Barrels, just seemingly not by the authors of the original Iron Chests mod.

**Comment by Roadhog360** [comment]
They are not by the og author, no.
